Doc/conf.py

```diff
@@ -254,6 +254,15 @@
 # Options for the link checker
 # ----------------------------
 
+linkcheck_allow_redirect = {
```

Comment:
```suggestion
linkcheck_allowed_redirects = {
```

re: https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-linkcheck_allowed_redirects

Reply:
Damn. I fixed that typo before and it came back to haunt me.